test/message/util.spec.ts

```diff
@@ -45,97 +44,102 @@ hhwl92GFuTXz4AEAqn4L+ayYgphejF/ZTRIseHPK+t521CT6NZKoVaHnTWQA
 -----END PGP PUBLIC KEY BLOCK-----`;
 
 test('it verifies a message with multiple signatures', async (t) => {
-    const publicKey1 = (await openpgp.key.readArmored(armoredPublicKey)).keys[0];
-    const publicKey2 = (await openpgp.key.readArmored(armoredPublicKey2)).keys[0];
+    const publicKey1 = await readKey({ armoredKey: armoredPublicKey });
+    const publicKey2 = await readKey({ armoredKey: armoredPublicKey2 });
     const { data, verified, signatureTimestamp, signatures, errors } = await verifyMessage({
-        message: createMessage('hello world'),
-        signature: await openpgp.signature.readArmored(detachedSignatureFromTwoKeys),
-        publicKeys: [publicKey1, publicKey2]
+        message: await createMessage('hello world'),
+        signature: await readSignature({ armoredSignature: detachedSignatureFromTwoKeys }),
+        verificationKeys: [publicKey1, publicKey2]
     });
-    t.deepEqual(data, openpgp.util.str_to_Uint8Array('hello world'));
```

Comment:
Should we always return `binary` data?

Reply:
No, I think we can change that, similarly to OpenPGP.js